package.json

```diff
@@ -53,7 +50,7 @@
     "format": "prettier --write \"**/*.{ts,tsx,js,scss}\" && npm run lint:fix",
     "format-check": "prettier --check \"**/*.{ts,tsx,js,scss}\"",
     "update-snapshot": "npm run test -- --updateSnapshot",
-    "pretest": "npm run format-check && npm run lint",
+    "pretest": "npm run format && npm run format-check && npm run lint",
```

Comment:
This was removed on purpose in https://github.com/nodejs/nodejs.dev/pull/871 because it ended up marking things as passed in CI even with checked in problems

Reply:
oh ok i'll close this then. Thanks for letting me know 👍